packages/axelar-wasm-std/src/threshold.rs

```diff
@@ -40,6 +40,14 @@ impl Fraction<Uint64> for Threshold {
     }
 }
 
+impl PartialOrd for Threshold {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        let self_normalized = self.numerator().full_mul(other.denominator());
+        let other_normalized = other.numerator().full_mul(self.denominator());
+        self_normalized.partial_cmp(&other_normalized)
```

Comment:
```suggestion
        
        self_normalized.partial_cmp(&other_normalized)
```

Reply:
Have some empty line. Rust doesn't use `return` for most of the cases, and an empty line before makes it look better.